queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -291,7 +300,9 @@ def reduce(self):
                     else:
                         self.message.message = "Run has been skipped in script"
             except Exception as exp:
-                with open(script_out, "a") as fle:
+                with open(create_log_path(file_name="Script.out",
+                                          log_directory=log_dir),
+                          "a") as fle:
                     fle.writelines(str(exp) + "\n")
```

Comment:
Why do we use logging to file here and logging to a handler elsewhere?

Reply:
I can't actually say as i didn't initially place logging here XD I am simply replacing the variable with the method call as such not changing any behaviour of reduce()
I decided to leave this in for the time being as to not change the structure of what reduce() is doing,  but will keep removing this in mind later down the line in the refactoring of reduce().